lib/apiRequest.js

```diff
@@ -9,7 +9,7 @@ async function apiRequest(method, path, params, body) {
     parameters.append(key, params[key]);
   });
 
-  const url = `${process.env.YAYA_API_URL}${path}${parameters.toString() ? '?' + parameters.toString() : ''}`;
+  const url = `${process.env.NODE_ENV === 'production' ? process.env.YAYA_API_URL : process.env.YAYA_TEST_API_URL}${path}${parameters.toString() ? '?' + parameters.toString() : ''}`;
```

Comment:
is the conditional right? looks like you dont append the path for production url